scripts/join/join_test.go

```diff
@@ -61,10 +62,11 @@ func TestJoinNetwork(t *testing.T) {
 	networkID := netconf.ID(*network)
 	haloTag := haloTag(t)
 	cfg := clicmd.InitConfig{
-		Network: networkID,
-		Home:    home,
-		Moniker: t.Name(),
-		HaloTag: haloTag,
+		Network:          networkID,
+		Home:             home,
+		Moniker:          t.Name(),
+		HaloTag:          haloTag,
+		HaloFeatureFlags: strings.Split(*featureFlags, ","),
```

Comment:
I think this test can automatically read the feature flags from the network manifest. No need to specify it via flags.